src/main/java/de/caritas/cob/userservice/api/admin/service/consultant/create/CreateConsultantDTOCreationInputAdapter.java

```diff
@@ -107,13 +107,23 @@ public boolean isTeamConsultant() {
   }
 
   /**
-   * Provdes the language formal flag.
+   * Provides the language formal flag.
    *
    * @return the language formal flag
    */
   @Override
   public boolean isLanguageFormal() {
     return isTrue(this.createConsultantDTO.getFormalLanguage());
   }
+  
+  /**
```

Comment:
sure. it is obvious. but the whole class has similar docs on every property. i don't know why, but i will stick to the convention, so that i don't diverge from existing impl.